packages/esm-patient-chart-app/translations/fr.json

```diff
@@ -85,6 +87,10 @@
   "other": "Autre",
   "pastVisitErrorText": "Erreur de visite passée",
   "pastVisits": "Visites passées",
+  "patientSummary":"résumé du patient",
+  "patient":"patiente",
+  "patientBreadcrumb": "patiente",
+  "Patient Summary Dashboard": "tableau de bord récapitulatif des patients",
   "priority": "Priorité",
```

Comment:
wouldn't it look better if these are sentence cased like **_Résumé du patient, Patiente_** or this can be specified in the config?

Reply:
Correct, @Jexsie. We generally try to use [sentence case](https://writer.com/blog/sentence-case/) for translation strings.

Reply:
Thanks @Jexsie and @denniskigen for the review!